frontend/src/app/components/transactions-list/transactions-list.component.html

```diff
@@ -111,7 +111,10 @@
                         <td style="text-align: left;" [innerHTML]="vin.inner_redeemscript_asm | asmStyler"></td>
                       </tr>
                       <tr *ngIf="vin.inner_witnessscript_asm">
-                        <td i18n="transactions-list.p2wsh-witness-script">P2WSH witness script</td>
+                        <td *ngIf="vin.prevout && vin.prevout.scriptpubkey_type == 'v1_p2tr'; else p2wsh" i18n="transactions-list.p2tr-tapscript">P2TR tapscript</td>
```

Comment:
> `i18n="transactions-list.p2tr-tapscript"`

i dont know how this localization exactly works, but i think this should do it

Reply:
or does it has to be added as a translation key somewhere?

Reply:
It is fully automatic. Just add a unique key for a unique string and it will be easier to recognize later in the translation-tool